### ISSUE TITLE
Add conda recipe

### ISSUE THREAD

**Issue by sarthakpati** [bayer-science-for-a-better-life/tiffslide#15]
### Describe the new feature:

It would be great to have a conda recipe so that it can be included with projects that have more complicated build processes (for example, using libraries that need C/C++ compilers).

### What is the current outcome? 

Have a recipe for `tiffslide` on the conda-forge channel.

### Is it backward-compatible? 

Yes, and it would be forward-compatible as well, because the conda bots can automatically fetch new sdists pushed to pypi.

**Comment by sarthakpati** [comment]
I am happy to work on this.

**Comment by sarthakpati** [comment]
This is ready to be merged into the conda channel: https://github.com/conda-forge/staged-recipes/pull/17932

**Comment by ap--** [comment]
Hi @sarthakpati 

Thanks for offering to work on this :heart: 
I didn't have the conda package very high on the priority list, since tiffslide is pure-python so far, and all dependencies were already in conda.

But I agree, it's nice to have it available as a pkg.
If you could add @sdvillal and me as maintainers to the conda-forge feedstock that would be great.

Thank you so much for your help.
Cheers,
Andreas :smiley:

**Comment by sarthakpati** [comment]
Hi @ap--,

Done, you both will need to leave a comment in https://github.com/conda-forge/staged-recipes/pull/17932 to ensure you get added on the team.

Cheers,
Sarthak

**Comment by sarthakpati** [comment]
Once this is merged, I can put in a PR detailing the recipe! 😄
